packages/protocol/contracts/team/airdrop/AirdropVault.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+/// @title ERC20Airdrop
+/// @notice Contract for managing Taiko token airdrop for eligible users.
+/// @custom:security-contact security@taiko.xyz
+contract AirdropVault is Ownable {
```

Comment:
Will we use indeed this vault contract or will it be a simple, dedicated hot/cold wallet ? I thought it will be the latter (in live mainnet).

Reply:
Ok so on mainnet it will be stored in the airdrop contract itself.
Given the design change i'd more comfortable if we try it out with dummy tokens, dummy addresses, everything end-to-end before we going really live.